examples/pypi-custom-registry/pixi.toml

```diff
@@ -0,0 +1,23 @@
+[project]
+name = "pypi-custom-registry"
+version = "0.1.0"
+description = "Add a short description here"
+authors = ["Tim de Jager <tim@prefix.dev>"]
+channels = ["conda-forge"]
+platforms = ["osx-arm64"]
+
+[dependencies]
+python = "3.12"
+
+[feature.alternative.pypi-options]
+index-url = "https://pypi.tuna.tsinghua.edu.cn/simple/"
+[feature.alternative.pypi-dependencies]
```

Comment:
Does this require a definition of that dependency here to be seen in the resolution?

Reply:
Sorry I don't understand what you mean :)?

Reply:
Ah yeah, I meant to say, can't you just define the PyPi dependencies once?

Reply:
Sure, then it goes into the default feature

Reply:
At the time I made the example 'no-default' was not yet a thing but I could use that now 😀